src/reconstruction/ppmc_cuda.cu

```diff
@@ -1,1277 +1,681 @@
 /*! \file ppmc_cuda.cu
  *  \brief Functions definitions for the ppm kernels, using characteristic
  tracing. Written following Stone et al. 2008. */
-#ifdef CUDA
-  #ifdef PPMC
```

Comment:
Why don't you need the PPMC def?

Reply:
Because we can compile the kernels just fine without it, they just won't be used anywhere.

Reply:
Generally my approach has been that unless an ifdef is absolutely necessary I don't include it (or remove it if I'm refactoring)

Reply:
Alright. I guess we don't really live in a world where compile time matters anymore :)